packages/testsweets/example/lib/ui/main/main_view.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:example/app/app.router.dart';
+import 'package:example/ui/main/main_viewmodel.dart';
+import 'package:example/ui/post/post_view.dart';
+import 'package:example/ui/todo/todo_view.dart';
+import 'package:flutter/material.dart';
+import 'package:stacked/stacked.dart';
+import 'package:testsweets/testsweets.dart';
+
+class MainView extends StatelessWidget {
+  const MainView({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final navigatorObserver = TestSweetsNavigatorObserver();
```

Comment:
To make this better we can implement the singleton pattern for the `TestSweetsNavigatorObserver`.

This way we can use 

```dart
TestSweetsNavigatorObserver.instance.setBottomNavIndex

// In the main file we can us

TestSweetsNavigatorObserver.instance
```

Since we don't need anything that's async we can construct the private local static instance of the observer in the get property `instance` if it's null.